### PR TITLE
close applications

### PR DESCRIPTION
**Pull request by karenhaining**
Change the message to say apps are closed this year

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -52,9 +51,10 @@ <h2 class="join-card-header">Students</h2>
         <!-- Watch a recording of Tuesday's info session <a target="_blank" href="https://washington.zoom.us/rec/share/TqObsDuGSoE1q2tf_mraey2ySYLIc4RNWvhhcphPnc0dyif9GkP9KxKaTywlSAND.XVmCDv8fnZCEO9-F?startTime=1664930306000">here</a>. -->
       </p>
```
Comment (manforowicz) [inline join.html:52]: Maybe add a `<br><br>` before the `<p>` to add some extra spacing.

Comment (manforowicz) [reply join.html:52]: Other then that it looks good!

Comment (manforowicz) [reply join.html:52]: But I just saw the Slack message, so up to you!

**Comment by AlnisS** [comment]
Based on some discussion, we will keep applications open in the current state, and will work on an updated application process moving forwards.

The immediate next step here is to update the wording on the page to say something to the effect of "Since we are close to the end of the university's year, new members will not have as much support in onboarding and getting started with projects." We may still have some folks join this spring or summer, but will likely need more independence.

Closing this for now, but we can re-open it if we choose to close applications.

Thanks!

**Pull request closed without merging**